3rdparty/peg-0.1.15/src/peg.peg-c

```diff
@@ -1,4 +1,4 @@
-/* A recursive-descent parser generated by peg 0.1.13 */
+/* A recursive-descent parser generated by peg 0.1.15 */
```

Comment:
so did this actually update the dependacny version? We should note that inthe dependancy tables in buildscripts I guess.

Reply:
This is not a runtime dependency, not even a build dependency. It's only needed when you want to regenerate the `math.pc` file from the `math.peg` grammar (which we do manually).

And I only updated `peg` to a version that was clearly used before to (re)generate the `math.pc` file.

Reply:
I see. thanks.